godotproject/Menus/HighScoreScreen/HighScoreScreen.gd

```diff
@@ -71,11 +71,8 @@ func _step_thonk_animation():
 		$ThonkSmallSFX.play()
 
 func _play_final_sound():
-	if final_idx >= num_scores_saved:
-		# If we didn't get a high score, play a sad sound :(
-		$SadSFX.play()
```

Comment:
Did you delete the .wav file that this audiostream player plays?

Reply:
Yep! "no_high_score.wav" was removed 👍